lamindb/models/has_parents.py

```diff
@@ -43,12 +43,12 @@ def _query_relatives(
         model = records[0].__class__
         frontier_ids = {r.id for r in records}  # type: ignore
 
-    if attr == "records":
-        attr_filter = "type__id__in"
-    elif attr == "children":
+    if attr == "children":
         attr_filter = "parents__id__in"
-    else:
+    elif attr == "parents":
         attr_filter = "children__id__in"
+    else:
+        attr_filter = "type__id__in"
```

Comment:
if it is the same for everything else (is this correct @falexwolf ?), then why not keep `Literal["children", "parents", "records"]` and just pass `"records"`?

Reply:
See all the examples in this diff!